### DISCUSSION TITLE
Add method pay - Dotpay

### DISCUSSION THREAD

**Discussion by jlukas99** [woocommerce/woocommerce#45071] [WooCommerce, Blocks, and Site Editing]
https://github.com/dotpay/WooCommerce2

Please add this method pay <3

**Comment by Aljullu** [comment]
Hi @jlukas99, thanks for sharing your feedback!

_WooCommerce Dotpay payment module_ is not developed by us, so we can't directly update it to add support for the Cart and Checkout blocks. I recommend you to [contact their developers](https://github.com/dotpay/WooCommerce2/issues) and encourage them to add support for the blocks. You can suggest them to take a look at the docs, which include information about [integrating payment methods with Cart and Checkout blocks](https://github.com/woocommerce/woocommerce-gutenberg-products-block/blob/trunk/docs/extensibility/payment-method-integration.md).

In the meanwhile, you can check the list of payment methods that already support the Cart and Checkout blocks in this documentation page:

https://docs.woocommerce.com/document/cart-checkout-blocks-support-status/#section-4
